### ISSUE TITLE
SPMC / Nvidia Shield - Unable to Play UHD Full Bluray

### ISSUE THREAD

**Issue by toro1977** [koying/SPMC#1089]
Hi guys,

When I try to play **UHD Full BluRay** Videos nothing happens. instead, when I play **UHD Remux** videos it works perfect, same with **1080p Full BluRay**.

Anyone going trough the same issue?

Thanks.

**Comment by ghost** [comment]
I don't think Kodi (or SPMC) yet supports Ultra HD Blu-ray structure. On Kodi,
 I think it might be solved when they merge support for Blu-ray menus,
 but since that requires Java, I am not sure if it will be added to SPMC.
 (https://github.com/koying/SPMC/issues/938)

Update: looking at threads on Kodi's forum, the team is waiting for libbluray to update (https://git.videolan.org/?p=libbluray.git) to support UHD.

**Comment by toro1977** [comment]
Thanks a lot.Waiting for an update then. :)

**Comment by riddleace** [comment]
I dont thinks its based at "Java" becaouse its just one mkv file...

**Comment by toro1977** [comment]
It's not a MKV File, that's the case of a Bluray REMUX, it's not the case, i'm talking about a FULL Bluray.

**Comment by riddleace** [comment]
ahh... ok... :-)

**Comment by BernardWG** [comment]
Hi Guys,

Does this same issue apply also for streaming 4K movies within SPMC ?  Since undeer Kodi Krypton 15.5 after about 30 minutes or so, the nVidida Shield TV return to Home Menu and the Kodi/SPMC is closed as well as my IPVanish connection. I have a 4K Philips TV  Television PHILIPS  4 K AmbiLight  Philips 49PUS7101  (with automatic UHD regcognition)
Thanks

**Comment by toro1977** [comment]
I use SPMC always so YES, no way to Play 4K Full Bluray movies as far as i know.

**Comment by BernardWG** [comment]
Yes. That is what I figured. Kodi as well as SPMC crash after about hour or
so. Wonder who is going to solve this issue. Koying/SPMC or nVidia ...

Why else having 4K Box and 4K TV ?

Thanks for your conment.

Bernard

Op 14 nov. 2017 15:47 schreef "toro1977" <notifications@github.com>:

**Comment by ghost** [comment]
Libbluray 1.0.2 released with support for playing UHD, now integrated into Kodi 18 nightly. Here's to hoping @koying will choose to integrate it too. (no BD-J menu support - though again, given complications of Java on Android TV, shouldn't expect that anyway)

**Comment by dabl2** [comment]
Agree, would very much like to see support for this in SPMC
